client/src/main/java/jp/co/soramitsu/iroha/java/TransactionBuilder.java

```diff
@@ -117,14 +137,48 @@ public TransactionBuilder setQuorum(int quorum) {
     return this;
   }
 
+  public TransactionBuilder setSettingValue(String key, String value) {
```

Comment:
hm, but the library can be used to construct genesis blocks as well, maybe just make a note in Javadoc for that and proceed with a proper implementation?

Reply:
Good point.